samples/Sentry.Samples.Uwp/App.xaml.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Sentry.Samples.Uwp
+{
+    sealed partial class App : Application
+    {
+        public App()
+        {
+            _ = SentrySdk.Init(new SentryOptions() { Dsn = "https://80aed643f81249d4bed3e30687b310ab@o447951.ingest.sentry.io/5428537", Debug = true });
+            InitializeComponent();
+            Suspending += OnSuspending;
```

Comment:
I wonder if we have a way to hook into this automatically, from within `Sentry` I mean, when the target is `uap..`

Reply:
there are three events that we can hook (Suspending, EnteredBackground and LeavingBackground)